src/app/shared/utils/layout/post-it-layout.ts

```diff
@@ -24,7 +24,7 @@ export class PostItLayout {
 
   protected layout: any;
 
-  protected insertingElementsAtIndex: number = 2;
+  protected insertingElementsAtIndex: number = 0;
```

Comment:
Are you sure with that? I recently changed it, because it adds post-its to end of layout and I think this is not very good behaviour.

Reply:
Well, it specifies the index the new elements should be added at.
It was 2, because add and import were indexes 0, and 1. Now they
are replaced by buttons, so 0 is similiar to inserting at the start.

Didn't it work now as expected, since you're asking?

Reply:
It's okay then.